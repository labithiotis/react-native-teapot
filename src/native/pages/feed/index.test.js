import React from 'react';
import { shallow } from 'enzyme';
import { Feed } from './index';

describe('Feed', () => {
  it('should render correctly', () => {
    const props = global.defaultProps({
      navigation: {
        state: {
          routeName: 'feed',
        },
      },
      screenProps: {
        feed: {
          error: null,
          loading: false,
          data: [],
        },
        locale: {
          pages: {
            feed: {
              back: 'test',
            },
          },
        },
      },
    });
    const component = shallow(<Feed {...props} />);
    expect(component).toMatchSnapshot();
  });
});
