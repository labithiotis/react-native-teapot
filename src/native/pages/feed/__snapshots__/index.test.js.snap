// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feed should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#000000",
        "flex": 1,
      },
      Object {
        "backgroundColor": "#239a05",
      },
    ]
  }
>
  <ScrollViewMock
    style={
      Array [
        Object {
          "flex": 1,
          "padding": 20,
        },
      ]
    }
  />
</View>
`;
